esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -42,6 +44,8 @@ for more details on the boards see mlrs-wifi-bridge-boards.h
 // Board
 // un-comment what you want
 //#define MODULE_GENERIC
+//#define MODULE_ESP32_WROOM32
```

Comment:
should be MODULE_EPS32_DEVKITC_V4, shouldn't it

Reply:
Done (although with correct *ESP32*).